src/components/inputlonganswer/InputLongAnswer.js

```diff
@@ -8,7 +9,27 @@ import {
 } from "./InputLongAnswerCSS";
 
 const InputLongAnswerComponent = (props) => {
-    const getInputVariantStyles = (p) => {
+    const [text, setText] = useState('');
+    const [errorText, setErrorText] = useState('');
+
+    const changeHandler = (e) => {
```

Comment:
@ajo01 could you let me know if this state actually gets updated from the textarea onChange on your end? It wasn't working on my end when I tried

Reply:
the state is getting updated. The problem is every time the text state changes the styled components are getting rerendered too so those components go out of focus. Moving the styled components above the InputLongAnswerComponent will fix this issue